### DISCUSSION TITLE
عدم سازگاری با ورژن ۱.۸.۳

### DISCUSSION THREAD

**Discussion by saeed3322** [iranxray/hope#103] [General]
این اسکریپت با ورژن ۱.۸.۳ سازگار نیست و error میده. در حالیکه xray config رو به حالت دیفالت برمیگردونم ورژن ۱.۸.۳ کار میکنه. پنل مورد استفاده کافکا هست.

**Comment by Phoenix-999** [comment]
لطفآ ایرادات و مشکلات را در قسمت Issues  بگذارد 
این مقاله ممکن هست سوال شما را جواب بده https://github.com/iranxray/hope/issues/101

**Comment by Phoenix-999** [comment]
@squall297

این مقاله ممکن هست سوال شما را جواب بده https://github.com/iranxray/hope/issues/101****
